### PR TITLE
Epic 7 Billing - update OrgBilling views

### PR DESCRIPTION
**Pull request by ackernaut**
This is a WIP for #1368 updating the organization billing views.

To be merged after invoice pagination and fixes.

Test Plan TBD

### CODE REVIEW

**Comment by ackernaut** [comment]
@mattkrick this branch is ready for your magic! See:

- Orgs list has Pro badge
- Org header has Pro badge
- Add Squeeze™ and Danger™ panels to Org Billing view

Let me know if ya need anything else!

File: src/universal/modules/userDashboard/components/OrgBilling/OrgBilling.js [hunk 1]
```diff
@@ -49,18 +50,22 @@ class OrgBilling extends Component {
...
+        {/* Plan Squeeze: show when !isPaid */}
+        <OrgPlanSqueeze />
+        {/* Credit Card: show when isPaid */}
         <Panel label="Credit Card Information">
           <div className={css(styles.infoAndUpdate)}>
             <div className={css(styles.creditCardInfo)}>
               <FontAwesome className={css(styles.creditCardIcon)} name="credit-card" />
               <span className={css(styles.creditCardProvider)}>{brand || '???'}</span>
-              <span className={css(styles.creditCardNumber)}>•••• •••• •••• {last4 || '••••'}</span>
+              <span className={css(styles.creditCardNumber)}>{'•••• •••• •••• '}{last4 || '••••'}</span>
```
[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/modules/userDashboard/components/OrgBilling/OrgBilling.js]: On both GitHub and Atom, strings of text that are not wrapped as a string have a funky formatting that I do not prefer. They are ambiguous to the tags etc. around them. Also, unwrapped copy might have the equivalent of JavaScript keywords and they get highlighted (like “do” and “let” and “if”). It is just a preference for me. I wouldn’t say it has to be a rule.

I could probably hack Atom but when I see files on GitHub it has the same effect, at least for keywords.

screenshot 2017-10-12 10 20 19
screenshot 2017-10-12 10 20 49

[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/modules/userDashboard/components/OrgBilling/OrgBilling.js]: I think it feels more explicit and clean that it is literally a JSX block of string (may have the terminology wrong).

[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/modules/userDashboard/components/OrgBilling/OrgBilling.js]: I admit sometimes seeing the variables in the template strings can be hard at a quick glance.

[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/modules/userDashboard/components/OrgBilling/OrgBilling.js]: Another note, justing “real” punctuation for copy means we don’t have to escape characters.

File: src/universal/components/Tag/tagBase.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import ui from 'universal/styles/ui';
```
[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/components/Tag/tagBase.js]: Yeah, I see your point. I guess I was thinking “things relative to tag” in `/Tag` but that might not always be true. I think we branch out with card styles, for example.

I’m ok doing some refactoring, but a little at a time, so not urgent/priority IMO. I just refactored my brain out on buttons and inputs.

File: src/universal/modules/userDashboard/components/OrgBilling/OrgBilling.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mattkrick) [reply src/universal/modules/userDashboard/components/OrgBilling/OrgBilling.js]: Makes sense to me!

**Pull request merged**